src/Entities/ExpOrb.h

```diff
@@ -44,9 +44,14 @@ class cExpOrb :
 
 	// tolua_end
 
+	/** Split reward into small values */
+	static void Split(int a_Reward, std::vector<int> & a_SplitReward);
```

Comment:
I think a better signature would be to return the `std::vector` directly, rather than using it as an out-param:
```cpp
static std::vector<int> & Split(int a_Reward);
```